src/software/thunderscope/thunderscope_main.py

```diff
@@ -317,6 +339,7 @@
             estop_mode=estop_mode,
             estop_path=estop_path,
             enable_radio=args.enable_radio,
+            sslvision_address=args.sslvision_multicast_address,
         ) as robot_communication:
```

Comment:
As part of the `--run_blue`/`--run_yellow` paths here, we should _optionally_ launch the GC context manager if `--launch_gc` is passed. Here's an example of how a context manager could be launched optionally:
https://github.com/UBC-Thunderbots/Software/blob/501df3656fea62eee7cee8fbc87fe24f4ac72030/src/software/thunderscope/thunderscope_main.py#L432-L444

Reply:
resolved a long time ago.